src/runtime-rs/crates/shim/src/bin/main.rs

```diff
@@ -142,7 +144,11 @@ fn real_main() -> Result<()> {
         Action::Delete(args) => {
             let mut shim = ShimExecutor::new(args);
             let rt = get_tokio_runtime().context("get tokio runtime")?;
-            rt.block_on(shim.delete())?
+            rt.block_on(shim.delete())?;
+
+            // XXX: end tracing when containerd asks kata to stop
+            // XXX: stop here to trace deletion process
+            trace_end();
```

Comment:
This function is called in a new process, seems it will not end the former global tracer.